2-Browsers/Week1/homework/ex2-aboutMe/index.js

```diff
@@ -8,4 +8,24 @@
    `list-item`.
 ------------------------------------------------------------------------------*/
 
-// TODO add your JavaScript code here.
+document.body.style.fontFamily = 'Arial, sans-serif';
+
+const nickname = document.getElementById('nickname');
+const parentNickname = nickname.parentElement;
+const newName = document.createTextNode('Veronika');
+parentNickname.replaceChild(newName, nickname);
+
+const favFood = document.getElementById('fav-food');
+const parentFavFood = favFood.parentElement;
+const newFood = document.createTextNode('Sushi');
+parentFavFood.replaceChild(newFood, favFood);
+
+const hometown = document.getElementById('hometown');
+const parentHometown = hometown.parentElement;
+const newTown = document.createTextNode('Moscow');
+parentHometown.replaceChild(newTown, hometown);
+
+const lis = document.getElementsByTagName('li');
```

Comment:
It is good practice to have nice and easy to read variable names. So instead of calling it lis, just type list